### PR TITLE
[Bugfix-2.0.x] Updated german translation

### PR DESCRIPTION
**Pull request by Ludy87**
### Description

Lutz Lehmann (FB):

» "Komposition - Deppenleerzeichen" - remove
» Grammar - changed

Ludy:

» Spelling - changed

### CODE REVIEW

File: Marlin/src/lcd/language/language_de.h [hunk 1]
```diff
@@ -135,17 +135,17 @@
 #define MSG_UBL_OUTPUT_MAP_HOST             _UxGT("Ausgabe für Host")
 #define MSG_UBL_OUTPUT_MAP_CSV              _UxGT("Ausgabe für CSV")
 #define MSG_UBL_OUTPUT_MAP_BACKUP           _UxGT("Externe Sicherung")
-#define MSG_UBL_INFO_UBL                    _UxGT("UBL Info ausgeben")
+#define MSG_UBL_INFO_UBL                    _UxGT("UBL-Info ausgeben")
 #define MSG_UBL_EDIT_MESH_MENU              _UxGT("Netz bearbeiten")
 #define MSG_UBL_FILLIN_AMOUNT               _UxGT("Menge an Füllung")
 #define MSG_UBL_MANUAL_FILLIN               _UxGT("Manuelles Füllen")
-#define MSG_UBL_SMART_FILLIN                _UxGT("Intelligente Füllen")
+#define MSG_UBL_SMART_FILLIN                _UxGT("Intelligentes Füllen")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_de.h]: Menu items must be 19 characters or fewer.
```
1234567890123456789|0
Intelligentes Fülle|n
```

**Comment by Ludy87** [comment]
German words be very long 😅

**Comment by AnHardt** [comment]
Only if you try to, brute force, translate everything. How about "Fachsprache"? Or "Denglisch"?
F.e. "Smart" is completely understandable for the target group today.
We don't have to translate to satisfy a "Deutschlehrer" - we have to translate to be understood by the target group.
Even when I began with programming Marlin to solve some problems in the German translations - mainly the Umlaut-problem - I wouldn't use that if i'd be paid for it. It's about that bad as "Word V1 Deutsch" - completely un-understandable - overtranslated.

**Comment by Ludy87** [comment]
@AnHardt 
I have to contradict you, Denglish or even English is the wrong way. Then there is no need for translation and you can save time.

**Comment by Ludy87** [comment]
Show this commit: https://github.com/MarlinFirmware/Marlin/commit/8bf596bd4cb6879eb50994d497b30acf84082395#diff-35ae64c5e89b5357ff0562d553782a3d

**Comment by thinkyhead** [comment]
Thanks for keeping abreast of the message name change. It's possible I might have forgotten to update it here.

**Comment by thinkyhead** [comment]
The translation debate is interesting.
If I could have one wish, it would be to stop capitalizing nouns.

**Comment by Ludy87** [comment]
Really? i wish me camelcase

**Comment by thinkyhead** [comment]
I guess that would help with words like Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz 
 and Donaudampfschifffahrtselektrizitätenhauptbetriebswerkbauunterbeamtengesellschaft.

**Comment by Ludy87** [comment]
beautiful words :-D

**Comment by AnHardt** [comment]
Noun chaining is great. But at this size not very readable. Smuggling in some '-' helps.
To describe this things in english I'd need several sentences.

**Comment by thinkyhead** [comment]
To be fair, there are also tiny German words that require volumes to explain. Start with "Geist."

**Comment by AnHardt** [comment]
"Geist"? Has a multitude of meanings. In almost all cases you can see from the context what could be meaned. Luckily in English you have, some well suting words to seperate the cases. Where: Ghost, Brain, XXX-alcohol, spirit are likely to be the most common. But it's hard to be complete. Somehow, the common thing is, they are all kind of 'volatile' ('flüchtig' 'unfassbar') - but that's far more philosophy than a simple translation.

**Pull request merged**